ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/button/button.tsx

```diff
@@ -106,13 +106,14 @@ type RootProps = {
   inText?: boolean
   buttonType: buttonTypeEnum
   fadeUntilHover?: boolean
+  gaseous: boolean
```

Comment:
Seems like this file would be good to split up as well (presentational/container)

Reply:
Why so?